pkg/destroy/openstack/openstack.go

```diff
@@ -224,7 +224,8 @@ func deleteServers(opts *clientconfig.ClientOpts, filter Filter, logger logrus.F
 		err = servers.Delete(conn, server.ID).ExtractErr()
 		if err != nil {
 			// Ignore the error if the server cannot be found and return with an appropriate message if it's another type of error
-			if _, ok := err.(gophercloud.ErrDefault404); !ok {
+			var gerr gophercloud.ErrDefault404
+			if !errors.As(err, &gerr) {
```

Comment:
Since you are not using gerr here in the if block, isn't it better to use errors.Is

Reply:
@abhinavdahiya 
Afaik, errors.Is checks if two errors are equal (i.e. err == gerr), but gophercloud generates new errors of `ErrDefault404` type for each request. It means that we have to compare their types (ErrDefault404 or not ErrDefault404).
If you know how I can use errors.Is here, please tell me.

Reply:
If you think As as is more appropiate, please explain that in the commit message. i'm fine as long as the reasoning is described.